src/main/java/com/oltpbenchmark/ThreadBench.java

```diff
@@ -113,6 +113,7 @@ private Results runRateLimitedMultiPhase() {
 
         // long measureStart = start;
 
+        long start_ts = System.currentTimeMillis();
         long start = System.nanoTime();
```

Comment:
Maybe we should make naming more explicit in general? I.e., startTimestampInMs, startTimestampInNs or something, the naming right now is a bit adhoc.